app/design/frontend/base/default/template/razorpay/extensions/firecheckout/payments.phtml

```diff
@@ -0,0 +1,66 @@
+<?php $helper = Mage::helper('razorpay_payments'); ?>
+<?php if ($helper->isRazorpayEnabled()) : ?>
+    <?php $code = Razorpay_Payments_Model_Paymentmethod::METHOD_CODE ; ?>
+    <script type="text/javascript">
+        var formId = 'firecheckout-form';
+        var paymentIdField = 'razorpay_payment_id';
+        var nativeButton = $$('#review-buttons-container .btn-checkout').first();
+
+        // Change native extension place order button onclick
+        nativeButton.setAttribute('onclick', 'processRazorpayOrder()');
+
+        // Click extension "place order" button
+        var processNativeExtensionOrder = function(){
+            nativeButton.show();
+
+            checkout.save();
+        };
+
+        var showErrorMessage = function(message){
+            alert(message);
```

Comment:
Do we have to use alert here? Doesn't firecheckout provide us with something better, for eg?